src/k-provider/ovp/response-types/kaltura-external-captions-list.js

```diff
@@ -0,0 +1,49 @@
+//@flow
+import ServiceResult from '../../common/base-service-result';
+
+const ASSET_ID_URL_INDEX: number = 10;
+
+export default class KalturaExternalCaptionsList extends ServiceResult {
```

Comment:
again, we have a convention to call the response types in the name of the BE types.
I assume it's `KalturaCaptionAssetListResponse`

Reply:
it's not always the case. (KalturaPlaybackContext) - but i will change